core/src/main/scala/com/evolutiongaming/kafka/flow/kafka/KafkaModule.scala

```diff
@@ -1,83 +1,80 @@
 package com.evolutiongaming.kafka.flow.kafka
 
-import cats.effect.Blocker
-import cats.effect.Clock
-import cats.effect.ConcurrentEffect
-import cats.effect.ContextShift
-import cats.effect.Resource
-import cats.effect.Timer
+import cats.effect.{Blocker, Clock, ConcurrentEffect, ContextShift, Resource, Timer}
 import cats.syntax.all._
-import com.evolutiongaming.catshelper.FromTry
-import com.evolutiongaming.catshelper.LogOf
-import com.evolutiongaming.catshelper.ToFuture
-import com.evolutiongaming.catshelper.ToTry
+import com.evolutiongaming.catshelper.CatsHelper._
+import com.evolutiongaming.catshelper.{FromTry, LogOf, ToFuture, ToTry}
 import com.evolutiongaming.kafka.flow.LogResource
-import com.evolutiongaming.kafka.journal.KafkaConfig
-import com.evolutiongaming.kafka.journal.KafkaHealthCheck
-import com.evolutiongaming.kafka.journal.RandomIdOf
 import com.evolutiongaming.kafka.journal.util.SkafkaHelper._
-import com.evolutiongaming.kafka.journal.{KafkaConsumerOf => JournalConsumerOf}
-import com.evolutiongaming.kafka.journal.{KafkaProducerOf => JournalProducerOf}
-import com.evolutiongaming.skafka.consumer.AutoOffsetReset
-import com.evolutiongaming.skafka.consumer.{ConsumerConfig, ConsumerMetrics, ConsumerOf => RawConsumerOf}
-import com.evolutiongaming.skafka.producer.{ProducerConfig, ProducerMetrics, ProducerOf => RawProducerOf}
-import com.evolutiongaming.smetrics.CollectorRegistry
-import com.evolutiongaming.smetrics.MeasureDuration
+import com.evolutiongaming.kafka.journal.{
+  KafkaConfig,
+  KafkaHealthCheck,
+  RandomIdOf,
+  KafkaConsumerOf => JournalConsumerOf,
+  KafkaProducerOf => JournalProducerOf
+}
+import com.evolutiongaming.skafka.consumer._
+import com.evolutiongaming.skafka.producer.{Producer, ProducerMetrics, ProducerOf}
+import com.evolutiongaming.smetrics.{CollectorRegistry, MeasureDuration}
+import monocle.macros.syntax.lens._
```

Comment:
Do we really want to introduce a dependency on monocle macros into the library code?

Reply:
Why we should not?

Reply:
It will make library migration to Scala 3 harder eventually. We will also be forcing Monocle dependency on the users, while they might prefer using shapeless or Tofu lens instead.

The worst part, we will actually be using specific Monocle version and the library users might use a different one, which is not binary compatible with ours.

I would not care at all if dependency was added to end-user application, but libraries is a bit more sensitive about the unnecessary dependencies.

In this specific case, we can survive the pain of updating the case class manually.

Reply:
Then I'll drop `monocle` dependency at runtime & use old good `.copy(...)`